Runtimes/Core/core/CMakeLists.txt

```diff
@@ -373,6 +373,15 @@ if(NOT ANDROID AND NOT APPLE AND NOT LINUX AND NOT WIN32 AND UNIX)
   target_link_libraries(swiftCore PRIVATE "${EXECINFO_LIBRARY}")
 endif()
 
+check_compiler_flag(Swift "-Xfrontend -enable-client-retain-release" HAVE_Swift_EMIT_CLIENT_RETAIN_RELEASE_FLAG)
```

Comment:
This check will likely need to be generalized to all of Core, Overlay and Supplemental -- this may require a bit of staging, so it could make sense to land the find module first and wire it up in subsequent PRs.